Sudoku_Solver/Sudoku_Solver.java

```diff
@@ -0,0 +1,112 @@
+import java.io.*;
```

Comment:
Avoid importing the entire package in your class. You can use Ctrl+ shift+ o to manage your imports in eclipse.

Reply:
done